src/utils.jl

```diff
@@ -307,6 +307,33 @@ end
 
 Base.length(storage::DeletedVertexStorage) = length(storage.storage)
 
+"""
+    Computes the linear minimizer in the direction on the PrecomputedSet.
```

Comment:
```suggestion
Computes the linear minimizer in the direction on the `PrecomputedSet`.
```

Reply:
use `precomputed_set` instead of CamelCase, and describe in the docstring what precomputed_set is (a set of vertices?)